### PR TITLE
Update README to mention caveat when building for Android

### PR DESCRIPTION
**Pull request by albertvaka**

### CODE REVIEW

**Comment by albertvaka** [comment]
Is this good to merge @ThiefMaster?

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ find the SDK.
 $ cordova platform add android
 ```
 
+**Note:** You need to have Android build tools 29.0.2 installed *and not have installed any version greater than 29.0.2*. Due to a [bug](https://github.com/apache/cordova-android/issues/1290) in `cordova-anroid` < 10.0, which this project uses, it's not possible to specify the build tools version with `--gradleArg=-PcdvBuildToolsVersion=29.0.2`. Because of this, it will always try to use the latest installed and fail with "Build-tool is missing DX" if they are greater than 29.0.2.
```
Comment (ThiefMaster) [inline README.md]: Please wrap the line similar to the other lines in here to avoid overly long lines :)

**Comment by ThiefMaster** [comment]
I'm not particularly familiar with Android stuff, but I've asked one of my colleagues to have a look to see if you are suggesting makes sense. If yes I'll gladly merge it.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,11 @@ find the SDK.
 $ cordova platform add android
 ```
 
+**Note:** You need to have Android build tools 29.0.2 installed *and not have installed any version greater than 29.0.2*.
+Due to a [bug](https://github.com/apache/cordova-android/issues/1290) in `cordova-anroid` < 10.0, which this project uses,
```
Comment (ThiefMaster) [inline README.md]:
```suggestion
Due to a [bug](https://github.com/apache/cordova-android/issues/1290) in `cordova-android` < 10.0, which this project uses,
```

**Comment by albertvaka** [comment]
@ThiefMaster Friendly ping, let me know if this is good to merge or I'll close it.

**Pull request merged**